frontend/src/App.js

```diff
@@ -1,21 +1,15 @@
-import React from 'react';
-import Form from "./components/Form";
+import React from "react";
 import GameChoice from "./components/GameChoice";
 import GameList from "./GameList";
-
-
+import NavBar from "./components/NavBar";
```

Comment:
Doesn't this trigger an error because of the unused component ? Am I missing something ?